target/linux/mediatek/image/filogic.mk

```diff
@@ -1536,6 +1536,17 @@ define Device/routerich_ax3000-ubootmod
 endef
 TARGET_DEVICES += routerich_ax3000-ubootmod
 
+define Device/routerich_ax3000-v1
+  DEVICE_VENDOR := Routerich
+  DEVICE_MODEL := AX3000 v1.0
```

Comment:
Revision should go without minor version to `DEVICE_VARIANT`.

Reply:
Can you drop the minor version (`.0`) as well? It is referred without this two lines above this one as well as in the DTS name.